saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/README.md

```diff
@@ -11,37 +11,71 @@ In this example, we implemented and compared the performance of SVM and Naive Ba
+The CLEF image data is made available in different files, we have used the following files in this project 1) wlist.txt, 2) labels.txt, 3) features.txt, 4) spatial_rels mat (Matlab) files 5) training.mat and 6) testing.mat.
+
+`wlist.txt` file contains the vocabulary used for annotation, the format of the file is Id followed by label.
+
+29	branch
+
+30	bridge
+
+These Id are used in other files for referring the associated label.
+
+`label.txt` file contains the codes of segments / objects found in the image, the format of the file is ImageId followed by segment sequence number followed by segment code.
+  
+25	1	29
+
+25	2	60
+
+25	3	31
+
+means that segments / objects "1", "2" and "3" in image "25" (.jpg) have associated the labels 29 ("branch"), 60 ("cloud") and 31 ("building") respectively.
+
+`feature.txt` file contains visual features extracted for each segment / object of the image. The visual features were extracted using code from Peter Carbonetto [3] and are the following: "region area, width and height of the region, mean and standard deviation in the x and y axis, boundary/area, convexity, average, standard deviation and skewness in RGB and CIE-Lab color spaces, for a total of 27 attributes". the format of the file is as follow:
+
+112	1	0.4417593         0.775     0.8583333     0.5560012     0.6382412      0.656913     0.7427811    0.01397768     0.3359084      119.3438      111.6017      105.8651      67.57657      64.81628      66.64162     0.3635511     0.5000695     0.7093432      68.94983      1.267164      3.321504      17.48109      8.993122      8.798025    -0.0448975     0.1654533      1.945854	120
+112	2	0.0189062     0.127083     0.219444     0.932729      0.59748      0.11265     0.170704    0.0624426     0.322059      69.2314      76.0888      70.3459      32.4674      33.4975       32.245      1.57548      1.40818      1.56848      59.4284     -3.34258      1.99991      10.3577      2.65751      1.48654     0.785716    -0.310416      0.87871	204
+
+The file contains 30 columns, columns 1 and 2 are as in the `labels.txt` file, columns 3-29 are the values of the extracted visual features, the last column is as the third column in `labels.txt`. 
+
+`spatial_rels` is a directory containing information of spatial relationships in the images. These relations are stored in a matlab file, with the same name of the image from which they were extracted.  Each file contains three matrices namely topo, x_rels and y_rels. The matrice topo stores the topological relations (i.e. 1: adjacent and 2: disjoint), x_rels matice stores the direction relations with reference to the x axis (i.e. 3: x-aligned and 4: beside) and y_rels matrice stores the direction relations with reference to the y axis (i.e. 5: y-aligned, 6: above and 7: below). 
+
+
```

Comment:
I have seen this document. It does not answer my question exactly. The only thing the the term `computed`; so that means they are not humane labeled but computed based on segmentation masks?